tasks/deploy_get_stamp_from_route.yml

```diff
@@ -11,9 +11,7 @@
     api_version: "v1"
     namespace: "{{ project_name }}"
     kind: "Route"
-    label_selectors:
-      - app={{ app.name }}
-      - route_prefix={{ prefix }}
+    label_selectors: ["app={{ app.name }},route_prefix={{ prefix }},!acme.openshift.io/temporary"]
```

Comment:
curious why not use the normally list syntax?

Reply:
I have a syntax error when adding a new entry with `-!acme.openshift.io/temporary`. We alreay use this syntax here https://github.com/openfun/arnold/blob/master/tasks/manage_app.yml#L101

Reply:
Interesting. Maybe surrounding it with quotes would also work.

Reply:
It works using quotes.

Reply:
I changed with the previous syntax and I use quotes now.